README.md

```diff
@@ -23,21 +23,21 @@ You can then use Chronos:
 <?php
 require 'vendor/autoload.php';
 
-use Cake\Chronos\Chronos;
+use Cake\Chronos\DateTime;
 
-printf("Now: %s", Chronos::now());
+printf("Now: %s", DateTime::now());
 ```
```

Comment:
Do you think it will be confusing that stdlib DateTime is mutabke, but this library uses the same name for an immutable object? It will be hard at a glance to know which Datetime a piece of code is using.